app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,15 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.likes-link {
+  text-decoration: none;
+  color: black;
+}
+
+#comment_bubble {
```

Comment:
Avoid using id selectors<br>Selector `comment_bubble` should be written in lowercase with hyphens